apps/console/src/features/sms-providers/models/sms-providers.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface SMSProviderAPIResponseInterface {
+    name: string;
+    provider: string;
+    providerURL: string;
+    key: string;
+    secret: string;
+    sender: string;
+    contentType: string;
+    properties: SMSProviderPropertiesInterface[];
+}
+
+export interface SMSProviderPropertiesInterface {
+    key: string;
+    value: string;
+}
+export interface SMSProviderPropertiesObjectInterface {
+    [key: string]: string;
+}
+export interface SMSProviderInterface {
+
```

Comment:
```suggestion
```
Let's not keep spaces between attributes in the interface. Check in other places as well